src/System.Text.JsonLab/System/Text/Json/JsonWriterUtf8.cs

```diff
@@ -915,5 +909,66 @@ private int AddNewLineAndIndentation(Span<byte> buffer)
             }
             return offset;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private int ItemSeperatorSize()
+        {
+            return (_firstItem ? 0 : 1);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private int WriteItemSeperator(Span<byte> byteBuffer)
+        {
+            int idx = 0;
+            if (_firstItem)
+            {
+                _firstItem = false;
+            }
+            else 
+            {
+                byteBuffer[idx++] = JsonConstants.ListSeperator;
+            }
+
+            return idx;
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private int SpacingSize()
+        {
+            return (_prettyPrint ? 2 + (_indent + 1) * 2 : 0);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private int WriteSpacing(Span<byte> byteBuffer)
```

Comment:
Wouldn't the return values of these methods always equal what the "{X}Size()" methods?

WriteSpacing would result whatever SpacingSize does. So, why do we need to return an integer here?